jwql/website/apps/jwql/templates/jwqldb_table_viewer.html

```diff
@@ -12,23 +12,110 @@
     	<h2>Explore JWQL database tables through the web browser</h2><hr>
 
         <p align="justify">
-        This page provides users to interactively explore the JWQL database tables with Astropy Tables Javascript Viewer.  Simply select a table from the dropdown menu.
+        This page provides users to interactively explore the JWQL database tables.  Simply select a table from the dropdown menu.
```

Comment:
might change this to "this page allows users to..."

Reply:
Just made the change, thanks!